src/cmd/cmdgazebo.rb.in

```diff
@@ -361,6 +361,12 @@ class Cmd
       # Import the runGui function
       Importer.extern 'int runGui(const char *)'
 
+      # If playback is specified, set the gui config to load the playback
+      # gui config
+      if (options['playback'] != '')
+        options['gui_config'] = "_playback_"
```

Comment:
I think it would be good to print a warning to the user in case they specified `--gui-config`.

Or even better, don't change the `gui_config` if the user manually specified it. They may be wanting to playback with a specific config.

Reply:
I just added a condition to not utilize the playback config if the `--gui-config` flag is specified here 924af51600d0083aeadfd8b682c0b9d39a157bb2